homework/index.js

```diff
@@ -29,23 +31,160 @@
     return elem;
   }
 
+  function convertDateTime(dateTimeTxt) {
+    const dateTime = new Date(dateTimeTxt);
+    return dateTime.toLocaleString();
+  }
+
+  function sortRepositoriesByName(a, b) {
+    return a.name.localeCompare(b.name);
+  }
+
   function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+    ul.textContent = ' ';
+    const repoList = createAndAppend('li', ul, { class: 'rep-list' });
```

Comment:
Be consistent. You have a variable `repoList` but a class name `rep-list`. Better make that `repo-list`. (I gave you this feedback already in week 1).

Reply:
I corrected the class name, thanks @remarcmij